multichain-testing/test/fast-usdc/config.ts

```diff
@@ -0,0 +1,27 @@
+import type { IBCChannelID } from '@agoric/vats';
+
+export const oracleMnemonics = {
+  oracle1:
+    'cause eight cattle slot course mail more aware vapor slab hobby match',
+  oracle2:
+    'flower salute inspire label latin cattle believe sausage match total bless refuse',
+  oracle3:
+    'surge magnet typical drive cement artist stay latin chief obey word always',
+};
+harden(oracleMnemonics);
+
+export const makeFeedPolicy = (nobleAgoricChannelId: IBCChannelID) => {
+  return JSON.stringify({
```

Comment:
can the caller `JSON.stringify` it? It would be nice to `@returns { FeedPolicy }`.